stubs/Driver/CursorInterface.stub.php

```diff
@@ -4,6 +4,7 @@
 
 use Iterator;
 use MongoDB\BSON\Int64;
+use Traversable;
```

Comment:
I don't see this referenced anywhere below.

Reply:
Nice catch, removed